tests/test_tools.py

```diff
@@ -191,7 +191,25 @@ def test_get_package_info(monkeypatch, inputs, expected):
     assert actual_metadata == expected
 
 
-def test_compute_mu_using_xraydb():
-    sample, energy, density = "ZrO2", 17445.362740959618, 1.009
-    actual_mu = compute_mu_using_xraydb(sample, energy, density=density)
-    assert actual_mu == pytest.approx(1.252, rel=1e-4, abs=0.1)
+params_mu = [
```

Comment:
Could you please follow the guides provided here? https://gitlab.thebillingegroup.com/resources/group-wiki/-/wikis/Group-standards-for-pytest-and-docstrings

add higher level function, use descriptive variable names, no need to create extra variable of `params_mu`,  no need to add "user specific, etc."

Reply:
@yucongalicechen 

@sbillinge if you have also further suggestions to the gitlab doc, please suggest or modify directly

Reply:
gitlab doc looks pretty good to me.  Lot's of great examples to work from.